docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+  - [Cleanup](#cleanup)
+
+### Introduction
+
+This sample demonstrates:
+
+  - Using the WebLogic Image Tool to create a Docker image that contains a WebLogic install, a WebLogic Deploy Tool (WDT) install, a Java EE servlet application contained within a WDT archive, and a WebLogic domain that's defined using a WDT model file.
+  - Modifying the WDT model that's embedded within the Docker image using a WDT model file that's supplied using a Kubernetes config map.
+  - Defining a `domainHomeSourceType: FromModel` domain resource that references the WDT model image and the WDT config map.
+  - Deploying the model image, domain resource, model config map, and associated secrets that define user names, passwords, and URL values for the model and its domain resource.
+  - Deploying and accessing a Traefik load balancer that redirects HTTP protocol calls to its Java EE servlet application.
+
+Supported domain types:
+
+There are three types of domains supported by Model in Image: a standard `WLS` domain, an Oracle Fusion Middleware Infrastructure Java Required Files (`JRF`) domain, or a `RestrictedJRF` domain.
+
+The `JRF` domain path through the sample includes additional steps for deploying an infrastructure database and initializing the database using the Repository Creation Utility (RCU) tool. `JRF` domains may be  used by Oracle products that layer on top of WebLogic Server such as SOA and OSB. Similarly, `RestrictedJRF` domains may be used by Oracle layered products such as Oracle Communications products.
+
+### References
+
+To reference the relevant user documentation, see:
+ - [Model in Image User Guide]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}) user documentation
+ - [Oracle WebLogic Server Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling)
+ - [Oracle WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool)
+
+
+
+### Prerequisites for all domain types
+
+1. The `JAVA_HOME` environment variable must be set and must reference a valid JDK 8 installation. (`JAVA_HOME` is used by the WebLogic Image Tool.)
+
+2. Set a source directory environment variable `SRCDIR` that references the parent of the operator source tree. For example:
+
+   ```
+   mkdir ~/wlopsrc
+   cd ~/wlopsrc
+   git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+   export SRCDIR=$(pwd)/weblogic-kubernetes-operator
+   ```
+
+   For additional information about obtaining the operator source, see the [Developer Guide Requirements](https://oracle.github.io/weblogic-kubernetes-operator/developerguide/requirements/).
+
+3. Create a sample directory environment variable `SAMPLEDIR` that references this sample's directory:
+
+   ```
+   export SAMPLEDIR=${SRCDIR}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/
+   ```
+
+4. Create an empty, temporary working directory with 10g of space, and store its location in the `WORKDIR` environment variable. For example:
```

Comment:
Broken record... I've found setting up these values to feel unnecessary.  I really like that I *can* set them.

Reply:
Tom:   V1? or V2: Make make this to something nice in /tmp - maybe /tmp/model-in-image-sample/$USER/...   Print out the location of WORKDIR in all scripts that use it.

Reply:
Done.  Default is now /tmp/$USER/model-in-image-sample-work-dir

Reply:
`10g` should probably be `10 GB` @rosemarymarano ?